INSTALL.md

```diff
@@ -191,12 +191,17 @@ OR (left click ) wampmanager icon -> MYSQL -> my.ini
 In Linux it's typically located in /etc/mysql
 
     1.  Look for the following line:
+    sql-mode = STRICT_TRANS_TABLES,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
+    or sometimes it maybe sql_mode
 
     2.  Change it to:
 ```
         sql_mode="" (Blank)
 ```
 
+(XAMPP)
+ If you don't find this parametery in my.ini file, you should run server, open http://localhost/phpmyadmin/ , click on the "variables" tab, search for "sql mode" and then check that it is set to:"" (Blank)
```

Comment:
Some thoughts. :)
* 'parametery' -> 'parameter'
* 'in my.ini file' -> 'in the my.ini file'
* remove space after /phpmyadmin/
* 'search for "sql mode" and' -> 'search for "sql mode," and'
* maybe just 'set to ""'?